packages/cardpay-subgraph/schema.graphql

```diff
@@ -456,3 +458,48 @@ type TokenPair @entity {
   token1: Token!
   swaps: [TokenSwap]!             @derivedFrom(field: "tokenPair")
 }
+
+type RewardProgramRegistrationPayment @entity {
+  id: ID!
+  rewardProgramID: String!
+  admin: Account!
+  transaction: Transaction!
+  createdAt: BigInt!
+  prepaidCardPayment: PrepaidCardPayment!
+}
+
+type RewardeeRegistrationPayment @entity {
+  id: ID!
+  rewardProgramID: String!
+  rewardee: Account!
+  transaction: Transaction!
+  createdAt: BigInt!
+  prepaidCardPayment: PrepaidCardPayment!
+}
+
+type RewardSafe @entity {
+  id: ID!
+  rewardProgramID: String!
+  rewardee: Account!
+  safe: Safe!
```

Comment:
should there be a derived field here to account for all the claims made against the reward safe:
```
claims: [RewardeeClaim]! @derivedFrom(field: "rewardSafe")

Reply:
YES

Reply:
This makes sense